### PR TITLE
feat: few updates and repository reference syncs 

### PR DESCRIPTION
**Pull request by dextermallo**
## Proposed changes

- Add .gitignore and a PR template
- Update README for readability
- Add contribution reference

## PR Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/coreruleset/coreruleset/blob/v4.0/dev/CONTRIBUTING.md) documentation.
- [x] I have tested the changes locally.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+<!-- Github Tip: adding the text 'Fixes #<issue>' or 'Closes #<issue>' will automatically close the mentioned issue. -->
+
+## PR Checklist
+
+<!-- _Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._ -->
+
+- [ ] I have read the [CONTRIBUTING](https://github.com/coreruleset/coreruleset/blob/v4.0/dev/CONTRIBUTING.md) documentation.
+- [ ] I have tested the changes locally.
```
Comment (theseion) [inline .github/pull_request_template.md:12]: This doesn't make sense for documentation.

File: README.md [hunk 2]
```diff
@@ -1,50 +1,39 @@
...
+## For Contributors
 
-You can edit the documentation on your local system. The only thing you need is the latest [Hugo binary](https://gohugo.io/getting-started/installing/) for your OS (Windows, Linux, Mac): it’s that simple.
+Welcome! This page will guide you through the process of contributing to the CRS documentation. Before you start, please read our [Contribution Guideline](content/development/contribution_guidelines.md).
 
-**Important: You need Hugo _extended_ version >= 0.93.0.**
+### 1. Prerequisites
 
-## Cloning this repository
+To contribute to the CRS documentation, The only thing you need is the latest [Hugo binary](https://gohugo.io/getting-started/installing/) for your OS (Windows, Linux, Mac) **(Important: You need Hugo _extended_ version >= 0.93.0.)**.
```
Comment (theseion) [inline README.md:15]:
```suggestion
To contribute to the CRS documentation the only thing you need is the latest [Hugo binary](https://gohugo.io/getting-started/installing/) for your OS (Windows, Linux, Mac) **(Important: You need Hugo _extended_ version >= 0.93.0.)**.
```

**Review by theseion — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+<!-- Github Tip: adding the text 'Fixes #<issue>' or 'Closes #<issue>' will automatically close the mentioned issue. -->
+
+## PR Checklist
+
+<!-- _Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._ -->
+
+- [ ] I have read the [CONTRIBUTING](https://github.com/coreruleset/coreruleset/blob/v4.0/dev/CONTRIBUTING.md) documentation.
```
[the remark this answers is not in the document]
Comment (dextermallo) [reply .github/pull_request_template.md:11]: Hmm just my preference to keep it in sync. Lemme remove it

File: README.md (hunk 2 above)
Comment (dextermallo) [reply README.md]: Fixed in [f8a8b1d](https://github.com/coreruleset/documentation/pull/132/commits/f8a8b1d136f7c5ab726e33c8fab633d945b6dbfd).

File: .github/pull_request_template.md (hunk 1 above)
Comment (dextermallo) [reply .github/pull_request_template.md]: Fixed in [f8a8b1d](https://github.com/coreruleset/documentation/pull/132/commits/f8a8b1d136f7c5ab726e33c8fab633d945b6dbfd).

**Review by theseion — APPROVED** [review]

**Comment by theseion** [comment]
Thanks a lot @dextermallo, sorry for the delay.

**Pull request merged**